### PR TITLE
[API compatibility] add Alias : torch.zeros, paddle.Tensor.view

### PR DESCRIPTION
**Pull request by zhengshengning**
### PR Category

Operator Mechanism

### PR Types

Improvements

### Description

[paddle.Tensor.view](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.view.html)
[paddle.zeros](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/torch/torch.zeros.html)

| API Name | Param | Alias | details|
|------|------|------|--------|
| paddle.Tensor.view| *shape  | shape_or_dtype | 指定的维度。 PyTorch 参数 shape 既可以是可变参数，也可以是 list/tuple/torch.Size/dtype 的形式， Paddle 参数 shape_or_dtype 为 list/tuple/dtype 的形式。对于可变参数的用法，需要进行转写。|
| paddle.zeros| x,axis   | input,dim | 表示输出形状大小，PyTorch 以可变参数方式传入，Paddle 以 list 或 tuple 的方式传入，需要转写。|

**用法:**
```
paddle.view(x=tensor_x, shape_or_dtype=[-1, 1, 3], name=None)
tensor_x.view(paddle.float32) -> paddle.view(tensor_x, paddle.float32)
tensor_x.view(dtype=paddle.float32) -> paddle.view(tensor_x, dtype=paddle.float32)
tensor_x.view([-1, 1, 3]) -> paddle.view(tensor_x, [-1, 1, 3])
tensor_x.view(-1, 1, 3) -> paddle.view(tensor_x, -1, 1, 3)
tensor_x.view(size=[-1, 1, 3]) -> paddle.view(tensor_x, size=[-1, 1, 3])
```

**Other**
增加了单测

pcard-67164

### CODE REVIEW

File: python/paddle/tensor/creation.py [hunk 1]
```diff
@@ -1381,6 +1381,7 @@ def ones_like(
     )
 
 
+@SizeArgsDecorator()
```
Comment (zhangbo9674) [inline python/paddle/tensor/creation.py:1384]: 注意后续补充中英文文档

File: python/paddle/tensor/manipulation.py [hunk 2]
```diff
@@ -7334,6 +7335,7 @@ def as_strided(
 
 
 @dygraph_only
+@view_decorator()
```
Comment (zhangbo9674) [inline python/paddle/tensor/manipulation.py:7338]: 同上，注意后续补充api 的中英文文档

**Review by zhangbo9674 — APPROVED** [review]
